walrus/methods/task/task_template/task_template_launch_handler.py

```diff
@@ -90,7 +83,9 @@ def task_template_new_exam(session, task_template):
     We don't create tasks here, since a person starting a new Exam
     will create a new job and new tasks
     """
-
+    result = task_new.provision_root_tasks(
```

Comment:
do we need this here? Since we have provisioning for each examination here -> 
https://github.com/diffgram/diffgram/blob/81ad551e6458312c29dd4c85ecafb334cf47ad28/default/methods/task/task_template/task_template_apply.py#L147